c/include/libsbp/sbas.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2015 Swift Navigation Inc.
+ * Contact: Fergus Noble <fergus@swift-nav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+/*****************************************************************************
+ * Automatically generated from yaml/swiftnav/sbp/sbas.yaml
+ * with generate.py. Please do not hand edit!
+ *****************************************************************************/
+
+/** \defgroup sbas Sbas
+ *
+ * * SBAS data
+ * \{ */
+
+#ifndef LIBSBP_SBAS_MESSAGES_H
+#define LIBSBP_SBAS_MESSAGES_H
+
+#include "common.h"
+#include "gnss.h"
+
+
+/** Raw SBAS data
+ *
+ * This message is sent once per second per SBAS satellite. ME checks the
+ * parity of the data block and sends only blocks that pass the check.
+ */
+#define SBP_MSG_SBAS_RAW 0x7777
+typedef struct __attribute__((packed)) {
+  sbp_gnss_signal_t sid;             /**< GNSS signal identifier. */
+  u32 tow;             /**< GPS time-of-week at the start of the data block. [ms] */
+  u8 message_type;    /**< SBAS message type (0-63) */
```

Comment:
Why do we need to transmit message_type? Won't the message ID in the data field give this information?

Reply:
This is the SBAS Message Type, which is outside the 212 bit data field

Reply:
ok, got it thanks